pkg/iac/terraform/schema.go

```diff
@@ -48,5 +48,8 @@ var Schema = &hcl.BodySchema{
 		{
 			Type: "import",
 		},
+		{
+			Type: "removed",
+		},
```

Comment:
I tried to find this list of `hcl.BlockHeaderSchema` types in the hcl repo but had no luck. I assume it doesn't exist or else we could just import it? Updating this each time would mean running into this problem again when a new block gets added.

Reply:
It's [here](https://github.com/hashicorp/terraform/blob/main/internal/configs/parser_config.go#L315). We can't import this as the terraform code is in the internal package.